iloop_to_model/app.py

```diff
@@ -253,6 +253,8 @@ class Meta:
         return ModelsMessage(response={k: ModelMessage(
             model=JSONValue(v['model']),
             model_id=v['model_id'],
+            # if the growth rate is 0.0, venom detects the field as undefined
+            growth_rate=v['growth-rate'] + 1e-106,
```

Comment:
Creative fix 😄
I don't see why venom should skip serializing zero numbers (which was implemented [here](https://github.com/biosustain/venom/commit/0e147eb772aed0363ee20778c342589aadf6925d)). So FYI I suggested to [fix it in venom](https://github.com/biosustain/venom/pull/27), but if it's not accepted or takes too long then this trick works fine.